optaplanner-core/src/main/java/org/optaplanner/core/config/solver/termination/TerminationConfig.java

```diff
@@ -51,13 +53,15 @@
 
     private TerminationCompositionStyle terminationCompositionStyle = null;
 
+    @XmlJavaTypeAdapter(JaxbDurationAdapter.class)
```

Comment:
this makes me worry.
In XStream, Jackson, etc, modern versions always support java.time out of the box, entirely.
Are we using an old version of jaxb? Or is jaxb dead?

Reply:
Already discussed (see above).